src/styles/blocks/price.scss

```diff
@@ -0,0 +1,13 @@
+.card__price-text {
```

Comment:
Add a class 'price' as a parent element to card__price-text and card__price-value classes to follow BEM methodology.